cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,46 @@
 
 describe('Student Registration page', () => {
   before(() => {
-    
+    cy.visit("https://demoqa.com/automation-practice-form");
   });
```

Comment:
I would recommend you to create a separate function to generate user and use it in your tests